pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -286,18 +286,33 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(ctx context.Context,
 		return reconcile.Result{RequeueAfter: timeout}, nil
 	}
 
-	// Verify if the volume is accessible to Pacific cluster.
-	isAccessible := isDatastoreAccessibleToCluster(ctx, vc, r.configInfo.Cfg.Global.ClusterID, volume.DatastoreUrl)
-	if !isAccessible {
-		log.Errorf("Volume: %s present on datastore: %s is not accessible to all nodes in the cluster: %s",
-			volumeID, volume.DatastoreUrl, r.configInfo.Cfg.Global.ClusterID)
-		setInstanceError(ctx, r, instance, "Volume in the spec is not accessible to all nodes in the cluster")
-		// Untag the CNS volume which was created previously.
-		_, err = common.DeleteVolumeUtil(ctx, r.volumeManager, volumeID, false)
-		if err != nil {
-			log.Errorf("Failed to untag CNS volume: %s with error: %+v", volumeID, err)
+	if isPodVMOnStretchedSupervisorEnabled {
```

Comment:
`isPodVMOnStretchedSupervisorEnabled` gate will be enabled even when driver is not running on StretchedSupervisor ?

Can we make sure we execute this workflow when we have more than one AZ and this fss is enabled?

Reply:
Can you see if the latest commit addresses the ask here.